napari/layers/labels/labels.py

```diff
@@ -436,6 +437,25 @@ def color(self, color):
         self._color = colors
         self.color_mode = color_mode
 
+    def _is_default_color(
+        self, color: Dict[Optional[int], np.ndarray]
+    ) -> bool:
+        """Checks whether a color is equal to the default color value of the layer."""
+        if len(color) != 2:
+            return False
+
+        default_color = {
+            None: transform_color('black')[0],
+            self._background_label: transform_color('transparent')[0],
+        }
```

Comment:
this should get defined once and be a private attribute on the layer

Reply:
Done, thanks for the suggestions